pom.xml

```diff
@@ -713,6 +725,61 @@
       </build>
     </profile>
 
+    <profile>
+      <id>nightly</id>
+      <build>
+        <pluginManagement>
+          <plugins>
+            <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-enforcer-plugin</artifactId>
+            <executions>
+              <execution>
+                <id>enforce-plugin-versions</id>
+                <goals>
+                  <goal>enforce</goal>
+                </goals>
+              </execution>
+            </executions>
+          </plugin>
+          </plugins>
+        </pluginManagement>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-source-plugin</artifactId>
+            <executions>
+              <execution>
+                <id>attach-sources</id>
+                <goals>
+                  <goal>jar</goal>
+                </goals>
+              </execution>
+            </executions>
+          </plugin>
+        </plugins>
+      </build>
+      <repositories>
+        <repository>
+          <id>nightly-staging</id>
+          <url>http://mavenin.nuxeo.com/nexus/content/repositories/nightly-staging</url>
```

Comment:
not blocker (fixed by Maven and Apache) but a preferred URL is
https://mavenin.nuxeo.com/nexus/content/repositories/nightly-staging/
with HTTPS and an ending slash